pkg/util/exec-run.go

```diff
@@ -0,0 +1,43 @@
+package util
+
+import (
+	"io/ioutil"
+	"os/exec"
+
+	"github.com/golang/glog"
+)
+
+type Runner interface {
+	RunCombinedOutput(string, ...string) ([]byte, error)
+	RunStdoutPipe(string, ...string) ([]byte, error)
+}
+
+type RealRunner struct{}
+
+// the real runner for the actual program, actually execs the command
+func (r RealRunner) RunCombinedOutput(command string, args ...string) ([]byte, error) {
```

Comment:
Can we give an example of terminal commands, where RunCombinedOutput is useful or should be used

Reply:
this code file was extracted from pool watcher implementation. I guess this has been used for executing the zpool create and zpool status commands. And I am using it for executing 'touch' and 'truncate' commands in my code.